### ISSUE TITLE
problem with line break

### ISSUE THREAD

**Issue by HenrikEckermann** [haozhu233/kableExtra#215]
I have problems with getting the linebreak working as needed. If I use this code in Rmarkdown,

```{r results = 'asis'}
table_note <- "A long table note with some line breaks \n and more text"
kable(mtcars[1:5,1:4], booktabs = T, caption = "Some caption.", escape = T) %>%
  footnote(general = table_note, general_title = "Note.", footnote_as_chunk = T, threeparttable = T)
```

I get:  

<img width="415" alt="table-ex" src="https://user-images.githubusercontent.com/22709846/41191169-88a638e8-6beb-11e8-9fb0-7c1be2fa3a69.png">

Another "issue": If I use `threeparttable = T`, I can get automatic linebreaks in alignment with the table but then I can not determine linebreaks myself +  there is a larger space between the general note and the beginning of the note, thus: I would need to use threepartables for all tables to stay at least consistent. I found that if I edit the .tex file by putting the whole table note into one item like this:

```
\begin{tablenotes}[para]
\item \textit{Note.} Computed after listwise deletion (upper diagnonal) or pairwise deletion (lower diagonal). * \textit{p} < .05. ** \textit{p} < .01. *** \textit{p} < .001
\end{tablenotes}
```
This works for me. Still I cannot add linebreaks manually or I at least did not find out how for the table notes. I also tried experimenting with `\makecell{}`but did not find a way yet.

**Comment by haozhu233** [comment]
@HenrikEckermann You are right. I guess makecell just doesn't work for table footnotes. I will think through it. Thanks for reporting!

**Comment by pauschae** [comment]
Are there any updates on this issue? I am having the same problem

**Comment by fthielen** [comment]
Same here. Would be nice to be able to add a line break in `footnote()`. I have a long footnote when I use `footnote_as_chunk = T` and this automatically extends the last column of the table.

**Comment by hnguyen19** [comment]
I am having the same problem as fthielen.
